jraft-core/src/main/java/com/alipay/sofa/jraft/storage/snapshot/SnapshotExecutorImpl.java

```diff
@@ -611,9 +611,9 @@ boolean registerDownloadingSnapshot(final DownloadingSnapshot ds) {
                 // this RPC.
                 saved = m;
                 this.downloadingSnapshot.set(ds);
-                result = false;
+                result = true;
             } else if (m.request.getMeta().getLastIncludedIndex() > ds.request.getMeta().getLastIncludedIndex()) {
-                // |is| is older
+                // |ds| is older
```

Comment:
This comment is error,  the `is` is installing snapshot.

Reply:
Is `line 625` also wrong? My understanding is that `|xx|` means `xx` is a variable. But there is no variable `is` in code.

Reply:
`m` is `is`, it was renamed someday before, but the comment was not changed.